src/Compilers/CSharp/Portable/Syntax/Syntax.xml

```diff
@@ -910,6 +910,36 @@
       <summary>Creates a LiteralExpressionSyntax node.</summary>
     </FactoryComment>
   </Node>
+  <Node Name="FieldExpressionSyntax" Base="ExpressionSyntax">
+    <Kind Name="FieldExpression"/>
+    <Field Name="Token" Type="SyntaxToken">
+      <Kind Name="FieldKeyword"/>
+      <PropertyComment>
+        <summary>SyntaxToken representing the field keyword.</summary>
+      </PropertyComment>
+    </Field>
+    <TypeComment>
+      <summary>Class which represents the syntax node for a field expression.</summary>
+    </TypeComment>
+    <FactoryComment>
+      <summary>Creates a FieldExpressionSyntax node.</summary>
+    </FactoryComment>
+  </Node>
+  <Node Name="ValueExpressionSyntax" Base="ExpressionSyntax">
+    <Kind Name="ValueExpression"/>
+    <Field Name="Token" Type="SyntaxToken">
+      <Kind Name="ValueKeyword"/>
+      <PropertyComment>
+        <summary>SyntaxToken representing the value keyword.</summary>
+      </PropertyComment>
+    </Field>
+    <TypeComment>
+      <summary>Class which represents the syntax node for a value expression.</summary>
+    </TypeComment>
+    <FactoryComment>
+      <summary>Creates a ValueExpressionSyntax node.</summary>
+    </FactoryComment>
```

Comment:
Alternatively, we could use LiteralExpressionSyntax, just like we use that for boolean expressions with `true` and `false` keyword.  but i'm not sure how compiler would feel about fallout of these being thought of as literals, so i think it's totally ok to not do that.

Reply:
Will leave as is.